lib/versioned/^4.0.0-alpha.1/index.js

```diff
@@ -42,10 +45,10 @@ function execute(opts, env) {
       return logTasksSimple(gulpInst.tree());
     }
     if (opts.tasks) {
-      var tree = {
-        label: 'Tasks for ' + chalk.magenta(tildify(env.configPath)),
-        nodes: gulpInst.tree({ deep: true }),
-      };
+      var tree = {};
+      tree.label = isString(config.description) ? config.description :
```

Comment:
It's not enough to check `isString`.  We don't want to print an empty string description